demo-forms/mrdt-demo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <h:head>
+    <h:title>MRDT Demo</h:title>
+    <model>
+      <instance>
+        <mrdt delimiter="#" id="mrdt_demo">
```

Comment:
Without editing the `id` from `mrdt_demo` to `mrdt`, I couldn't get the image to show up on report detail. Not sure if this is common knowledge or if this form is supposed to be `plug-and-play` functional or not.